tests/dotnet-lib-6/Dockerfile.glibc.runner

```diff
@@ -0,0 +1,12 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0-focal as builder
+
+# This dockerfile takes the glibc build of dotnet library test, and append a custom test runner to run the test DLLs
+
+# Copy the runner project to /runner in the builder
+COPY runner /runner
+RUN cd /runner && dotnet build -c Release
+
+FROM hullcritical/dotnet-library-test:release-glibc-v6
```

Comment:
I know this is a copy of an existing Dockerfile used for dotnet tests current, but I'm curious what this `hullcritical` repository is, and where did you get it from?

Reply:
hullcritical is actually my personal docker hub account haha. The image is built by using tests/dotnet-lib-6/Dockerfile.glibc.

Reply:
Thanks for the clarification. How is Dockerfile.glibc built and uploaded to the repository (e.g. manually)?

Reply:
Manually, `docker publish`

Reply:
@asvrada  Do we need to use your personal docker hub account? Can we instead use Azure Container Registry

Reply:
It definitely makes more sense to use an official one. Is there doc on how to use that?

Reply:
Yes it would be preferrable to avoid personal repositories. Could you speak to @rs-- for using an ACR instead? Alternatively, if the build process for Dockerfile.glibc is not long then it could just be built and referred locally instead.

Reply:
Update: will merge this PR first, then replace all personal docker image in a followup PR

Reply:
Okay, that sounds fine!